client/allocrunner/networking_bridge_linux.go

```diff
@@ -134,8 +138,44 @@ func (b *bridgeNetworkConfigurator) Teardown(ctx context.Context, alloc *structs
 	return b.cni.Teardown(ctx, alloc, spec)
 }
 
-func buildNomadBridgeNetConfig(bridgeName, subnet string) []byte {
-	return []byte(fmt.Sprintf(nomadCNIConfigTemplate, bridgeName, subnet, cniAdminChainName))
+func buildNomadBridgeNetConfig(b bridgeNetworkConfigurator) []byte {
+	// Parse the internal template that generates the Nomad bridge interface.
+	// The template's parsablity is tested in networking_bridge_linux_test.go
+	// so any template issues should be caught by the test rather than at this
+	// point.
+	tmpl, err := template.New("cniConf").Parse(nomadCNIConfigTemplate)
+	if err != nil {
+		// Panic on error for catching issues in testing
+		panic(err)
+	}
```

Comment:
I think you can move this to the top of file so the template is parse just once at initialization, and the use [`Must`](https://pkg.go.dev/text/template#Must) to panic, like:

```go
var nomadBridgeTmpl = template.Must(template.New("cniConf").Parse(nomadCNIConfigTemplate))
```